lib/order.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/menu.rb'
+require './lib/verify.rb'
+
+class Order
+
+attr_reader :current_order, :current_menu, :item_no, :item_total, :collect_amount
+
+    def initialize(menu_klass: Menu)
+      @current_order = []
+      @current_menu = []
+      @menu_klass = menu_klass
+    end
+
+    def start_order(item, item_no)
+     current_menu << menu_klass.new
+     current_order << current_menu[0].menu.select {|k,v| k == item}
+     current_order[-1][:item_no] = item_no
+     current_order[-1][:item_total] = (current_order[-1][item] \
+     * current_order[-1][:item_no])
+
+    end
+
+    def next_item(item, item_no)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for next_item is too high. [20/15] (http://c2.com/cgi/wiki?AbcMetric)